### DISCUSSION TITLE
Micrio IIIF image quality degrading

### DISCUSSION THREAD

**Discussion by salger1** [thegetty/quire#905] [Get Help] [answered]
I am having a problem with my image quality degrading when I put IIIF images from [Micr.io](https://micr.io/) into my publication. It appears that the viewer is only downloading a single 1024x1024 tile of the image and isn't downloading any deeper zoom levels. I tested the viewer with IIIF images from other non-Micrio sources and they worked fine. 
 
You can see the IIIF image on Micrio here: https://dashboard.micr.io/#/nelson-atkins-museum/projects/iiif-project/starr/upload/edit/LoGYX/en
And the same file in our Quire site: https://wwwstag.nelson-atkins.org/starr1/Continental/Seventeenth-Century/F58-60-102/
 
I used this manifest and canvas:
    manifestId: https://iiif.micr.io/LoGYX/manifest
    canvasId: https://iiif.micr.io/LoGYX/canvas

I also tried using https://iiif.micr.io/LoGYX/info.json in the manifestId but it wouldn't load an image and just kept spinning. I tried with and without a src and poster in case it was pulling that image for some reason but that didn't help. 

The full code from my figures yaml file is below. Thanks for your help!!

```
# F58-60/102 - Petitot, Cardinal Mazarin
  - id: "2102.0"
    src: https://blobcdn.nelson-atkins.org/wpmediablob/Starr/F58-60-102_Petitot-PortraitOfCardinalMazarin_Recto.png
    manifestId: https://iiif.micr.io/LoGYX/manifest
    canvasId: https://iiif.micr.io/LoGYX/canvas
    poster: https://blobcdn.nelson-atkins.org/wpmediablob/Starr/F58-60-102_Petitot-PortraitOfCardinalMazarin_Recto.png
    caption: "Jean Petitot, *Portrait of Cardinal Mazarin*, ca. 1660, enamel
      on gold, sight&colon; 11&sol;16 x 9&sol;16 in. (1.8 x 1.4 cm), framed&colon; 1 5&sol;8 x 1 1&sol;2 in. (4.1 x 3.8 cm), Gift of Mr. and Mrs.
      John W. Starr and the Starr Foundation, Inc., F58-60/102"
    alt: "Portrait miniature of a light-skinned man wearing a red cassock before a
      brown background."
    download: true
```

**Comment by workergnome** [comment, marked as answer]
Just as a suggestion, it looks like the Micr.io manifests are not perfectly valid, at least when run through https://presentation-validator.iiif.io/.  That might be the issue here, if you're not seeing it with other manifests?

**Reply by salger1** [reply]
Thank you for the quick response. I reached out to Micrio about that and I'll update when I hear back.

**Reply by salger1** [reply]
Micrio responded and said "I found out what went wrong: the Micrio IIIF Presentation API was still using version 2.0, whereas Quire was expecting version 3.0. I've updated our service to 3.0, and now your original link works correctly with zooming features!" and the zoom looks like it is working. I'll let you know if I have further questions. Thank you for your help!

**Reply by Erin-Cecele** [reply]
Great news that you were able to fix the issue @salger1. Congrats on a beautiful publication. I know a lot of hard work went into it!And thank you for the help @workergnome.
